packages/components/src/core/styles/common/defaultTheme.ts

```diff
@@ -557,4 +730,20 @@ export interface Borders {
   warning: Border;
 }
 
-export const defaultTheme = createThemeAdaptor(defaultThemeOptions);
+/**
+ * Default theme, uses light mode with no option to change it.
+ *
+ * @deprecated Use the `theme` to get a flexible light/dark mode theme function
```

Comment:
Maybe we don't deprecate it, some folks may never be interested in light/dark so why complicate their lives?

Alternatively, deprecate but give the new one a default theme of "light". 🤔

Reply:
Great question! Yeah I also feel that maybe don't deprecate it and just default to "light" sounds less intrusive 😄 👍